examples/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/setuppayloadscanner/BarcodeFragment.kt

```diff
@@ -181,8 +183,17 @@ class BarcodeFragment : Fragment() {
 
     private fun handleInputQrCode(qrCode: String) {
         lateinit var payload: SetupPayload
+        var isManualPairingCode = false
```

Comment:
where do we use `isManualPairingCode` ? Looks write-only in this method, so in this case should we just not have this variable?

Reply:
I guess we should pass it as `isShortDiscriminator` in `CHIPDeviceInfo.fromSetupPayload`.

Reply:
Sorry, some codes were missing while trying to merge the codes. I added the code and modified some variable names.